### PR TITLE
improving installation procedure

### PR DESCRIPTION
**Pull request by cane-lagerwaard**

### CODE REVIEW

File: docs/backend/installing-magento2.md [hunk 1]
```diff
@@ -33,6 +34,29 @@ bin/magento admin:user:create \
   --admin-lastname='Deity'
 ```
 
+#### Connect Falcon to your Magento instance
+
+Configure Falcon Server to connect to your Magento instance.
+You can do so by changing the `default.json` file that resides in `<your-app-name>/server/config`
+
+> Note: Use your API admin user that you've just created here
```
Comment (Borales) [inline docs/backend/installing-magento2.md]:
```suggestion
> Note: Use your API admin user credentials that you've just created here
```

File: docs/backend/installing-magento2.md [hunk 2]
```diff
@@ -33,6 +34,29 @@ bin/magento admin:user:create \
...
+
+```json
+{
+  "api-magento2": {
```
Comment (Borales) [inline docs/backend/installing-magento2.md]: @cane-lagerwaard  it must be nested with `apis` key
(this relates to WP section too)


```json
{
  "apis": {
    "api-magento2": { ... }
  }
}
```

Comment (cane-lagerwaard) [reply docs/backend/installing-magento2.md]: `apis` key is already in default.json, I'm only referencing to those key:value pairs

Comment (cane-lagerwaard) [reply docs/backend/installing-magento2.md]: but having that in there would be technically more correct indeed

**Pull request merged**